java/vector/pom.xml

```diff
@@ -32,6 +32,19 @@
       <artifactId>arrow-memory-core</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.arrow</groupId>
+      <artifactId>arrow-memory-netty</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+<!--    <dependency>-->
+<!--      <groupId>io.netty</groupId>-->
+<!--      <artifactId>netty-buffer</artifactId>-->
+<!--    </dependency>-->
+<!--    <dependency>-->
+<!--      <groupId>io.netty</groupId>-->
+<!--      <artifactId>netty-common</artifactId>-->
+<!--    </dependency>-->
```

Comment:
?

Reply:
Thanks, this was a leftover, I have removed it here https://github.com/sirensolutions/arrow/pull/16/commits/333f1b8e6dfea9a7b268f81f474b89577981701f